x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -65,19 +72,21 @@ export const AllCasesList = React.memo<AllCasesListProps>(
       ...(!isEmpty(hiddenStatuses) && firstAvailableStatus && { status: firstAvailableStatus }),
       owner: hasOwner ? owner : availableSolutions,
     };
+    const [filterOptions, setFilterOptions] = useState<FilterOptions>({
```

Comment:
This is the second logic change in the PR. the previous hook was working both as a fetch data hook and state management. I removed the state management part from it and move it here.